test/topology_experimental_raft/test_tombstone_gc.py

```diff
@@ -198,3 +198,39 @@ async def verify_tombstone_gc(tombstone_mark, timeout=30):
 
             # test #3: the tombstones are cleaned up after the node is started again
             await verify_tombstone_gc(tombstone_mark)
+
+
+@pytest.mark.asyncio
+@pytest.mark.xfail(reason="Issue #21117")
+async def test_group0_state_id_failure(manager: ManagerClient):
```

Comment:
nit: I think it's not worth running this test in multiple modes, one (dev) should suffice.

Reply:
done